scripts/drone/rgm.star

```diff
@@ -333,10 +336,67 @@ def rgm_main_pipeline():
         rgm_main(),
     ]
 
+def rgm_promotion_pipeline():
+    """Promotion build pipeline.
+
+    Returns:
+      Drone pipeline.
+    """
+    promotion_trigger = {
+        "event": ["promote"],
+        "target": "rgm",
```

Comment:
Would as target "custom-build" perhaps be a more fitting name?

Reply:
i would say that's too specific, since it doesn't have to be a "custom" build, it can really be any package, custom or not :joy: 

maybe `publish` or `package`?

Reply:
changed it to `upload-package`